examples/call-record-aggregator/akka-cdr-ingestor/src/main/scala/carly/ingestor/CallRecordValidation.scala

```diff
@@ -32,9 +33,12 @@ class CallRecordValidation extends AkkaStreamlet {
   val right = AvroOutlet[CallRecord]("valid", _.user)
 
   final override val shape = StreamletShape(in).withOutlets(left, right)
-  final override def createLogic = new SplitterLogic(in, left, right) {
+  final override def createLogic = new RunnableGraphStreamletLogic() {
+    def runnableGraph = {
+      sourceWithOffsetContext(in).to(Splitter.sink(flow, left, right))
```

Comment:
This uses the newly deprecated method, right?

```suggestion
      sourceWithCommittableContext(in).to(Splitter.sink(flow, left, right))
```

Reply:
no, sourceWithOffsetContext is not deprecated. At the source you know it is `CommittableOffset`.